### PR TITLE
Move Save CSS button to sidebar

### LINKED ISSUE #104: Move or duplicate the Save button
To improve usability at `Appearance > Custom CSS` either add an additional save button above Revisions or move the current save button to that location.

@AlexGStapleton Which solution would you prefer?

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve  #104

### CODE REVIEW

**Comment by Misplon** [comment]
Perhaps try:

For the button:
```
font-size: 14px;
min-height: 40px;
```

For the submit class:
`padding-top: 0;`

And then let me know what you think.

**Comment by AlexGStapleton** [comment]
I think `padding-top: 1px;` on the `.submit` looks a little better due to the border SiteOrigin CSS toolbar. Thoughts?

**Comment by Misplon** [comment]
Sounds good.

File: css/admin.less [hunk 1]
```diff
@@ -74,6 +74,15 @@
 	*:last-child {
 		margin-bottom: 0;
 	}
+
+	p.submit {
+		padding-top: 1px
```
Comment (Misplon) [inline css/admin.less:79]: `;` needed here to compile the Less

**Pull request merged**